src/pages/index.tsx

```diff
@@ -33,7 +33,7 @@ export default IndexPage
 
 const featureQuery = groq`
 {
-  'featureCallOut': ${digitalGardeningQuery}
+  'featureDigitalGardening': ${digitalGardeningQuery},
```

Comment:
`featureCallOut` was not descriptive and did not reflect the content being loaded on the index page.

This sets a consistent naming pattern (e.g. `featureDeveloperPorfolios`) for when we start adding more "features" on the home page